cps/tasks/download.py

```diff
@@ -54,6 +56,9 @@ def run(self, worker_thread):
                 last_progress_time = datetime.now()
                 fragment_stuck_timeout = 30  # seconds
 
+                self.message = f"Downloading {self.media_url_link}..."
+                if self.live_status == "was_live":
+                    self.message += f" (This may take longer than expected due to the video was live for {self.duration})"
```

Comment:
Explain more?

(I don't quite yet understand this section!)

Reply:
If the video is a "formerly live" video, this lets the user know that. Live videos are generally long, the duration is mentioned in the message then.

Reply:
Ah, nothing unusual here, ok:

We just need to fix the English grammar explaining this a bit better!